From afc503b054456bb66ef903e3a94b672a35addedb Mon Sep 17 00:00:00 2001
From: TonyLiu <Bo.Liu@windriver.com>
Date: Fri, 17 Oct 2008 15:00:28 +0800
Subject: [PATCH] imx_mx3_Kconfig

MX3 arch kernel configuration option. HAVE_CLK config enable use of
kernel clk interface. we don't realize gpiolib required functions for MX3 

Signed-off-by: Tonyliu <Bo.Liu@windriver.com>
---
 arch/arm/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/Kconfig b/arch/arm/Kconfig
index 70dba16..7e45223 100644
--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -415,11 +415,11 @@ config ARCH_MV78XX0
 
 config ARCH_MXC
 	bool "Freescale MXC/iMX-based"
+	select HAVE_CLK
 	select GENERIC_TIME
 	select GENERIC_CLOCKEVENTS
 	select ARCH_MTD_XIP
 	select GENERIC_GPIO
-	select ARCH_REQUIRE_GPIOLIB
 	help
 	  Support for Freescale MXC/iMX-based family of processors
 
-- 
1.5.5.1

